Frends.RabbitMQ.Publish/Frends.RabbitMQ.Publish/Definitions/Enums.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Frends.RabbitMQ.Publish.Definitions;
+
+/// <summary>
+/// Data input types.
+/// </summary>
+public enum InputType
+{
+#pragma warning disable CS1591 // self explanatory.
+    String,
+    ByteArray
+#pragma warning restore CS1591 // self explanatory.
+}
+
+/// <summary>
+/// Authentication methods.
+/// </summary>
+public enum AuthenticationMethod
+{
+    /// <summary>
+    /// Connect with URI.
+    /// </summary>
+    /// <example>amqp://foo:bar@localhost:1234</example>
```

Comment:
No need to provide examples in Enums. Examples should be provided in the class which is using this enum as a property.